oem/ibm/requester/dbus_to_file_handler.cpp

```diff
@@ -421,6 +423,71 @@ void DbusToFileHandler::newFileAvailableSendToHost(const uint32_t fileSize,
     }
 }
 
+void DbusToFileHandler::sendFileAckWithMetaDataToHost(
+    uint16_t fileType, uint32_t fileHandle, uint8_t fileStatus,
+    uint32_t fileMetaData1, uint32_t fileMetaData2, uint32_t fileMetaData3,
+    uint32_t fileMetaData4)
+{
+    info("sendFileAckWithMetaDataToHost fmd1={FMD1} fmd2={FMD2} fmd3={FMD3} "
```

Comment:
please improve tracing, all over the patch.

Reply:
info(sending acknowledgement With MetaData To Host where metadata1:{DATA1} metadata2: {DATA2})

Reply:
Done